### ISSUE TITLE
Propagating entities ADD them to an ArchitecturalMap instead of REPLACING the previous content

### ISSUE THREAD

**Issue by NicolasAnquetil** [moosetechnology/MooseIDE#1173]
Open an ArchitecturalMap with some entities
propagate some other entities
They are added to the visualization and I think they should replace the existing visualization

Or we make a setting to choose

What should the default behavior be ?

**Comment by ClotildeToullec** [comment]
The default behavior in all browsers is to replace entities.
If you want more entities, you propagate more.

**Comment by badetitou** [comment]
I agree with Clotilde for now. And I do not have a specific boring use case that would make me change the current behavior
